mteb/models/repllama_models.py

```diff
@@ -85,9 +89,17 @@ def encode(
     ) -> np.ndarray:
         batch_size = 16 if "batch_size" not in kwargs else kwargs.pop("batch_size")
         all_embeddings = []
-        prompt = self.get_prompt_name(self.model_prompts, task_name, prompt_type)
+        prompt_name = self.get_prompt_name(self.model_prompts, task_name, prompt_type)
+        prompt = self.model_prompts.get(prompt_name)
+
         if prompt:
-            sentences = [f"{prompt}{sentence}".strip() for sentence in sentences]
+            if prompt_name == "queries":
```

Comment:
shouldn't this check for prompt_type?

Reply:
Good question. I am not as familiar with this part of the prompting code. Is prompt_type either query or passage? If so, I can switch it.